### ISSUE TITLE
Postgres 14.11 isn't compiled with “USE_FLOAT8_BYVAL” (missing linux/arm64 build)

### ISSUE THREAD

**Issue by FarisZR** [docker-library/postgres#1195]
> The database cluster was initialized with USE_FLOAT8_BYVAL but the server was compiled without USE_FLOAT8_BYVAL.
tested on an arm oracle cloud instance.

This issue started with 14.11 release, reverting to 14.10 fixes it, both alpine and non-alpine version are affected.
https://github.com/FarisZR/Server/commit/1030867c1bf62e49832bebc1cccaca93439b9487

It seems that the 14.11 image wasn't compiled for linux/arm64? Only a linux/arm/v6 image is available on docker hub, maybe that helps.

https://hub.docker.com/layers/library/postgres/14.11-alpine/images/sha256-90bcf5fca0b62305b1c5418e562b092d482554e1ffe0ebee8f73b9820855bcba?context=explore

**Comment by yosifkit** [comment]
So, yeah this sounds like `dockerd`/`contianerd` did the regular fallback of pulling the "architecture of best fit" for the image and that currently happens to be the `arm32v6` image on an `arm64v8` host.

The PostgreSQL 14.11 images were added to the builder queue with https://github.com/docker-library/official-images/pull/16220, but we also merged https://github.com/docker-library/official-images/pull/16234 and caused a lot of dependent image rebuilds so it has been delayed in getting built on all architectures. The `arm64v8` builder is especially backed up at the moment; hopefully the `14.11` image will finish soon.
